core/io/http_client.h

```diff
@@ -116,7 +115,6 @@ class HTTPClient : public Reference {
 	};
 
 	enum Method {
-
```

Comment:
There's also empty lines before the closing `};` for some enums. It would be great if we managed to fix those with a script too.

Reply:
This will work for closing braces at the start of lines, but not for indented closing braces.

    sed -z -i 's/\x0A\x0A\x7D/\x0A\x7D/g' "$f"